Sources/Kitura/RouterResponse.swift

```diff
@@ -23,9 +23,25 @@ import KituraContracts
 
 // MARK: RouterResponse
 
-/// Router response that the server sends as a reply to `RouterRequest`.
+/**
+ The RouterResponse class is used to define and work with the response that will be sent by the `Router`.
+ It contains and allows access to the HTTP response code (e.g. 404 notfound), the HTTP `Headers`
```

Comment:
404 Not Found rather than 404 notfound seems to be the usual way of referring to this.